sg14-error-codes.html

```diff
@@ -591,6 +625,11 @@
     The programmer may head far down this "garden path" under the assumption that his goal of a non-zero "ok" code is attainable;
     but we on the Committee know that it is <i>not</i> attainable. We should save the programmer some time and some headaches,
     by explicitly reserving error-code <code>0</code> in the standard.
+    
+    <!-- I personally think that any system_error2 should copy the design of NT kernel error codes which have four subdomains,
+    success, success with information, failure with information, failure where the top bit is success/failure, so simply testing codes
+    for < 0 means "it failed". Obviously we don't encode that into the error code itself, but rather into the v2 design of categories,
+    especially if we can somehow get them to carry payload -->
```

Comment:
I don't know enough to interpret this comment. If there's a good online resource for quickly learning about NT error codes, it might be good to stick it in this paper so that readers (and writers ;)) can get informed.